src/peft/tuners/lora/model.py

```diff
@@ -348,27 +358,27 @@ def _set_adapter_layers(self, enabled=True):
     def enable_adapter_layers(self):
         self._set_adapter_layers(enabled=True)
 
-    def _get_active_adapter(self) -> str:
-        active_adapter = None
+    def _get_active_adapters(self) -> str:
+        active_adapters = None
         for module in self.model.modules():
             if isinstance(module, LoraLayer):
-                active_adapter = module.active_adapter
+                active_adapters = module.active_adapters
 
-        if active_adapter is None:
+        if active_adapters is None:
             raise ValueError(
                 "Something went wrong, no active adapter could be found, please report the issue on GitHub"
             )
-        return active_adapter
+        return active_adapters
```

Comment:
Wait, is this correct? This only returns a single adapter, right (e.g. "default")? Therefore, code like `for active_adapter in self._get_active_adapters()` iterates over the characters of that string.

Reply:
Hello, in the function it is getting `module.active_adapters` which is a list.

Reply:
Ah yes, you're right, sorry for the confusion. Then could you please update the type annotation, which is still `str`?

Reply:
Done